### DISCUSSION TITLE
Logos Round 3! [Deadline: Feb 16]

### DISCUSSION THREAD

**Discussion by ebullient** [commonhaus/foundation#69] [Consensus Building]
We're on to round 3! 

If you go back and look, we did end up clustering (first two rows) and clumping (a few variants of B). 

[Round 3 various sizes and context](https://docs.google.com/presentation/d/12WH8bufO0gx5_B9Lz5Hg-b0BJSWysbw1s8Xt--kWOg8/edit?usp=sharing), [large image 1](https://design.jboss.org/commonhaus/images/commonhaus_r3v1_large.png), [large image 2](https://design.jboss.org/commonhaus/images/commonhaus_r3v2_large.png), [large image 3](https://design.jboss.org/commonhaus/images/commonhaus_r3v3_large.png)

<img src="https://design.jboss.org/commonhaus/images/commonhaus_r3_comparison.png" width="85%" />

Remember: We still thinking about concepts here. More refinement to come. 

- Rank them in order of preference.
- (🚀, optional) Specific number if you have a favorite
- (👎, optional) Specific number if it is an absolute no

Constructive commentary is helpful.

**Comment by cealsair** [comment]
Here's is my order of preference:
- 1
- 3 
- 2

**Comment by gavinking** [comment]
1 is still my favorite :)

**Comment by kenfinnigan** [comment]
🚀 1 is my preference, then 3 and 2

**Reply by trustin** [reply]
Same here! 🚀 1 is my preference, then 3 and 2.

**Comment by sigrunixia** [comment]
With color added:
1 🚀
2
3 👎

No contest

**Reply by valentine195** [reply]
Agreed. 1 is easily the best to me, 2 a distant second.

**Comment by cowtowncoder** [comment]
Nice progress: any of these would be acceptable. But otherwise I think these are in my order of precedence so:

* 1
* 2
* 3

**Comment by aalmiray** [comment]
Here goes:

2
1
3

**Comment by nmcl** [comment]
1, 3, 2

**Comment by jkschneider** [comment]
Order of preference: 1, 2, 3.
